crates/libeq_wld/src/parser/fragments/material.rs

```diff
@@ -93,14 +30,18 @@ pub struct MaterialFragment {
     // Bit 3 ........ Set to 1 if the texture is masked and semi-transparent.
     // Bit 4 ........ Set to 1 if the texture is masked but not semi-transparent.
     // Bit 31 ...... Apparently must be 1 if the texture isn’t transparent.
-    pub transparency_flags: TransparencyFlags,
+    pub render_method: RenderMethod,
 
     /// This typically contains 0x004E4E4E but has also bee known to contain 0xB2B2B2.
     /// Could this be an RGB reflectivity value?
-    pub params2: u32,
+    /// RGBPEN %d, %d, %d
+    pub rgb_pen: u32,
+
+    /// BRIGHTNESS %f
+    pub brightness: f32,
 
-    /// Coordinate in the texture with the mask color
-    pub mask_color_coord: (f32, f32),
+    /// SCALEDAMBIENT %f
+    pub scaled_ambient: f32,
```

Comment:
The ASCII material definitions lead me to believe that these fields are `brightness` and `scaled_ambient` rather than the `mask_color_coord`. When comparing the ASCII defs to what's compiled into the wld file they match up perfectly. It's still possible I missed something here though! And this unfortunately leaves us without a mask color if this _is_ the case 😢.

Reply:
Haha I guess it was just luck then.  I thought it was the X,Y coord in the texture of where the key color was.  Perhaps that (or the color itself) is defined elsewhere...